webapp/package.json

```diff
@@ -43,7 +43,9 @@
     "enzyme-to-json": "3.4.3",
     "eslint": "6.0.1",
     "eslint-import-resolver-webpack": "0.11.1",
+    "eslint-plugin-cypress": "2.11.2",
```

Comment:
Is that really needed /here/?

Reply:
Right, this seems unrelated to the `webapp` folder. I think we need to move `webapp/e2e` to `e2e` at the root of the repo. Possibly `tests-e2e` to match what the server repo's convention